// Find Even/Odd 

import java.util.Scanner;;

public class Problem_26 {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the number: ");
        int n = sc.nextInt();

        if(n % 2 == 0){
            System.out.println("Even");
        } else {
            System.out.println("Odd");
        }

        sc.close();
    }
    
}
